causync.py

```diff
@@ -149,17 +179,25 @@ def get_basename(self, path):
 
         return basename
 
-    def get_lockfile_path(self):
+    def check_lockfiles(self):
+        """ check if logfiles exist for all source directories """
+        result = False
+        for path in self.src_abs:
```

Comment:
This seems to be wrong. If there are two sources, and the first has a lock while the second does not, then result is False. Maybe you wanted to use result |=. Please also log all lock files found.

Reply:
Thanks for noticing, this was a typo. :)